console/src/tasks.rs

```diff
@@ -114,8 +136,23 @@ impl State {
         if let Some(now) = update.now {
             self.last_updated_at = Some(now.into());
         }
+
+        if let Some(new_metada) = update.new_metadata {
+            let metas: HashMap<_, Metadata> = new_metada
```

Comment:
nit: i'd prefer either
```suggestion
        if let Some(new_metadata) = update.new_metadata {
            let metas: HashMap<_, Metadata> = new_metadata
```
or
```suggestion
        if let Some(new_meta) = update.new_metadata {
            let metas: HashMap<_, Metadata> = new_meta
```
as i feel like metadata is rarely, if ever, abbreviated as "metada"...